### PR TITLE
{math}[foss/2020a,intel/2020a] MUMPS v5.2.1

### PR DESCRIPTION
**Pull request by MaximeMoge**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by lexming** [comment]
Re-triggering tests

**Comment by zao** [comment]
Test report by @zao
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
freja - Linux ubuntu 18.04, Intel(R) Core(TM) i7-8700K CPU @ 3.70GHz, Python 2.7.17
See https://gist.github.com/5989fd85e1ac86ec2b3e8e4a3cf33fe2 for a full test report.

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
login2.cerberus.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6126 CPU @ 2.60GHz, Python 2.7.5
See https://gist.github.com/a0553478c7eedbf5cae92b97b0346007 for a full test report.

**Comment by migueldiascosta** [comment]
Test report by @migueldiascosta
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
sms - Linux centos linux 7.6.1810, AMD EPYC 7601 32-Core Processor, Python 2.7.5
See https://gist.github.com/1e813e584d1fe9d44be4937252a3dbf2 for a full test report.

**Comment by MaximeMoge** [comment]
Test report by @MaximeMoge
**FAILED**
Build succeeded for 0 out of 2 (2 easyconfigs in this PR)
tcn971.bullx - Linux RHEL 7.7, Intel(R) Xeon(R) CPU E5-2690 v3 @ 2.60GHz, Python 3.7.5
See https://gist.github.com/ca5e84704324a65433acebbe0e6fbff3 for a full test report.

**Comment by akesandgren** [comment]
Hmm the Hypre easyconfigs doesn't belong in thiis PR, please remove them

**Comment by MaximeMoge** [comment]
`eb --new-pr` is just not working with out setup, I will do a "normal" PR instead

**Comment by akesandgren** [comment]
For removing Hypre from this pr do eb --update-pr=xxx :Hypre-... --pr-commit-msg='Remove Hypre ...'

Use a good commit msg

File: easybuild/easyconfigs/m/MUMPS/MUMPS-5.2.1-foss-2020a-metis.eb [hunk 1]
```diff
@@ -0,0 +1,31 @@
+name = 'MUMPS'
+version = '5.2.1'
+versionsuffix = '-metis'
+
+homepage = 'http://graal.ens-lyon.fr/MUMPS/'
```
Comment (akesandgren) [inline easybuild/easyconfigs/m/MUMPS/MUMPS-5.2.1-foss-2020a-metis.eb]: Use https for the homepage

File: easybuild/easyconfigs/m/MUMPS/MUMPS-5.2.1-intel-2020a-metis.eb [hunk 2]
```diff
@@ -0,0 +1,31 @@
+name = 'MUMPS'
+version = '5.2.1'
+versionsuffix = '-metis'
+
+homepage = 'http://graal.ens-lyon.fr/MUMPS/'
```
Comment (akesandgren) [inline easybuild/easyconfigs/m/MUMPS/MUMPS-5.2.1-intel-2020a-metis.eb]: Use https for the homepage

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
login2.cerberus.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6126 CPU @ 2.60GHz, Python 2.7.5
See https://gist.github.com/501f06f009d603b30b1e1d829d617264 for a full test report.

**Comment by akesandgren** [comment]
Test report by @akesandgren
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
skalman.hpc2n.umu.se - Linux ubuntu 18.04, Intel(R) Core(TM) i7-8565U CPU @ 1.80GHz, Python 2.7.17
See https://gist.github.com/7c0f73055cda7fd49f0eb05dc8806d6a for a full test report.

**Comment by akesandgren** [comment]
Test report by @akesandgren
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
generoso - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/db95667550ecfa13a5b9095dfda1b923 for a full test report.

**Comment by akesandgren** [comment]
Going in, thanks @MaximeMoge!

**Pull request merged**